### ISSUE TITLE
500 Errors when folder doesn't exist within WatchedFiles attribute of iisnode web.config element

### ISSUE THREAD

**Issue by windhandel** [tjanczuk/iisnode#412]
Hi Tomasz,

I just found a bug within the iisnode watchedFiles web.config attribute.

If you put a Folder in the watchedFiles attribute that does not exist, then the application just throws a 500 error (likely because of the folder not existing).

e.g.watchedFiles="_.js;iisnode.yml;_.config;API\Public*;"

Assuming API\Public doesn't exist, you get 500 errors.

**Comment by rramachand21** [comment]
does it work if API\Public exists?

**Comment by windhandel** [comment]
Yes, this is just an FYI.  The workaround is to remove the "bad" paths from the watchedFiles. But it's a pretty easy copy/paste bug and it's quite hard to isolate in a fully filled out web.config file.

We had to binary search it within the web.config finally after spending an hour+ determining that was the source of the issue fairly randomly.

**Comment by rramachand21** [comment]
Ok, I will look into it.

**Comment by windhandel** [comment]
Thanks Ranjith!
